### PR TITLE
fix: maps selection for s2 cell id tag

### LINKED ISSUE #1952: Maps: fix issue with s2_cell_id selection when column is renamed
Maps works when provided with the below query:
```
import "experimental"
import "experimental/geo"
import "experimental/csv"
csv.from(url: "https://raw.githubusercontent.com/influxdata/influxdb2-sample-data/master/usgs-earthquake-data/all_week-annotated.csv")
  |> filter(fn: (r) => r._measurement == "earthquake" and (r._field =~ /mag|lat|lon/))
   |> pivot(rowKey:["_time","id"], columnKey: ["_field"], valueColumn: "_value")
  |> map(fn: (r) => ({ r with _value: r.mag, _field: "mag", s2_cell_id:  geo.s2CellIDToken(point: {lat: r.lat, lon: r.lon}, level: 10)}))
  |> experimental.group(columns: ["_field"], mode: "extend")
  |> drop(columns: ["lat", "lon"])
```
If we rename the column `s2_cell_id` to `palak_cell_id`, the dropdown to select the column to use for lat/lon calculation, breaks due to a duplicate keys in the dropdown object.

### PR DESCRIPTION
**Pull request by Palakp41**
Closes #1952 

Kapture 2021-07-09 at 14 29 53

### CODE REVIEW

File: src/visualization/types/Map/S2ColumnOptions.tsx [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const S2ColumnOptions: FC<Props> = ({properties, update, results}) => {
 
   const tags = findTags(results.table)
 
+  console.log('Tags: ', tags)
```
Comment (hoorayimhelping) [inline src/visualization/types/Map/S2ColumnOptions.tsx:21]: ✂️

File: src/visualization/types/Map/view.tsx [hunk 2]
```diff
@@ -222,6 +224,8 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
       ],
     }
   }
+
+  console.log('config: ', config)
```
Comment (hoorayimhelping) [inline src/visualization/types/Map/view.tsx:228]: probably just want to git checkout this file since the changes are `console.log` statements

**Review by hoorayimhelping — APPROVED** [review]
lgtm, once you clean up the console statements :)

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 61 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1960)

**Pull request merged**